programmers/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,19 @@
+function solution(s) {
+  let answer = "";
+  let arr = [];
+  let strArr = [];
+  for (let i = 0; i < s.length; i++) {
+    arr.push(s.charCodeAt([i])); // 아스키코드로 바꿔서 배열arr에 저장
+  }
+  arr.sort(function (a, b) {
+    return b - a;
+  }); // arr에 원소들 내림차순으로 정렬
```

Comment:
es6부터 도입된 화살표 함수라는 개념이 있습니다. function을 사용하지 않고 화살표로 함수임을 나타내주는 것인데요 찾아보시면 도움이 될겁니다.

또한 주석은 코드를 봐도 이 코드가 어떤 기능을 하는지 모를 때 사용해주시고 되도록 주석 없이도 알아보기 쉬운 코드를 작성하는 것이 좋습니다.

이 경우에는 주석이 필요 없어 보이네요 아 물론 나머지 주석두요

[arrow function](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

```suggestion
  arr.sort((a, b) => {
    return b - a;
  });
```